src/main/java/racingcar/domain/model/Race.java

```diff
@@ -0,0 +1,50 @@
+package racingcar.domain.model;
+
+import racingcar.global.ErrorMessage;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class Race {
+
+    private final RacingCars racingCars;
+    private final TryNumber tryNumber;
+
+    private Race(RacingCars racingCars, TryNumber tryNumber) {
+        this.racingCars = racingCars;
+        this.tryNumber = tryNumber;
+    }
+
+    public static Race create(RacingCars racingCars, TryNumber tryNumber) {
+        return new Race(racingCars, tryNumber);
+    }
+
+    public RacingCars startRace() {
+        return RacingCars.create(racingCars.moveAll());
+    }
```

Comment:
시작하는 것뿐만 아니라 계속해서 시도하고 있는 것이라면 try가 낫지 않을까요?
```suggestion
    public RacingCars tryRace() {
        return RacingCars.create(racingCars.moveAll());
    }
```